src/app/components/Footer/index.jsx

```diff
@@ -1,27 +1,119 @@
 import React from 'react';
 import styled from 'styled-components';
 import {
-  C_STORM,
+  C_GREY,
   C_WHITE,
   FF_NEWS_SANS_REG,
+  GEL_SPACING,
   GEL_SPACING_DBL,
 } from '../../lib/constants/styles';
 
 const StyledFooter = styled.footer`
-  background-color: ${C_STORM};
+  background-color: ${C_GREY};
   font-family: ${FF_NEWS_SANS_REG};
-  padding: ${GEL_SPACING_DBL}px;
+  padding: ${GEL_SPACING_DBL}px ${GEL_SPACING}px;
+`;
+
+const StyledList = styled.ul`
+  border-bottom: 1px solid ${C_WHITE};
+  display: grid;
+  grid-auto-flow: column;
+  grid-template-columns: repeat(2, 50%);
+  grid-template-rows: repeat(4, 50%);
```

Comment:
This currently works since it sets the number of grid rows to be 4 and we have 7 items. When we dynamically change the number of links in the footer (for non-News services) we will need to check the number of entries and update this value programmatically.